packages/interface/src/features/projects/components/ProjectItem.tsx

```diff
@@ -53,7 +54,7 @@ export const ProjectItem = ({
 
         <div className="line-clamp-2 h-10 text-sm text-gray-400">
           <Skeleton className="w-full" isLoading={isLoading}>
-            {metadata.data?.bio}
+            {removeMarkdown(metadata.data?.bio || "")}
```

Comment:
do you think it's necessary to remove markdown styling here?

Reply:
I removed it as it might break the card sizes if we display the markdown, and it might read weird if we only leave the string